src/main/java/com/github/technus/tectech/thing/metaTileEntity/multi/GT_MetaTileEntity_EM_EyeOfHarmony.java

```diff
@@ -1056,14 +1083,18 @@ private void drainFluidFromHatchesAndStoreInternally() {
         }
     }
 
-    private EyeOfHarmonyRecipe currentRecipe;
+    private static EyeOfHarmonyRecipe currentRecipe;
```

Comment:
This looks wrong to me, this will make all EOHs in the world share the same recipe.

Reply:
`parallelAmount` also shouldn't be static